resources/views/user/TambahDataPeriode.blade.php

```diff
@@ -27,7 +27,7 @@
                     Periode
                   </label>
                   <div class="col-md-9 col-sm-9 col-xs-12">
-                    <input type="text" class="form-control col-md-12 col-xs-12" name="namaperiode" required>
+                    <input type="text" class="form-control col-md-12 col-xs-12" name="namaperiode" required pattern="(/[a-z0-9]\s)(/[A-Z0-9]\s).{0,}+" title="Format penulisan periode yang benar, contoh : 2018/2019 Ganjil">
```

Comment:
kada berasa patternnya, sama ay kaya kd be pattern. coba tulis `&` aja tetap mau

Reply:
lwn validasi sagan edit datanya mana